### PR TITLE
Salesforce Adobe Target Actions Public Beta Changes

### PR DESCRIPTION
**Pull request by kdaswani**
### Proposed changes
Prepping changes ahead of Public Beta of Salesforce (Actions), Adobe Target Web and Adobe Target Cloud Mode. Public Beta will be 4/26 so we would like these deployed on the Tues, 4/26 deploy. Thanks!

### Merge timing
- On a specific date: April 26

### CODE REVIEW

File: src/connections/destinations/catalog/actions-adobe-target-cloud/index.md [hunk 1]
```diff
@@ -12,6 +11,9 @@ Segment offers two destinations for Adobe Target:
 - [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
 - [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)
 
+> info ""
+> This document is about a feature which is in beta. This means that the Adobe Target Cloud Mode destination is in active development, and some functionality may change before it becomes generally available.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-cloud/index.md:15]:
```suggestion
> The Adobe Target Cloud Mode destination is in beta and is in active development. Some functionality may change before it becomes generally available.
```

File: src/connections/destinations/catalog/actions-adobe-target-web/index.md [hunk 2]
```diff
@@ -2,16 +2,18 @@
...
+Adobe Target is the A/B testing and personalization component of Adobe Experience Cloud. Segment’s Adobe Target integration enables customers to send data from Segment to Adobe Target to create and update user profiles. You can leverage these profiles in Adobe Target to construct audiences and personalize onsite visitor experiences.
 
 Segment offers two destinations for Adobe Target:
 - [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
 - [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)
 
+> info ""
+> This document is about a feature which is in beta. This means that the Adobe Target Web destination is in active development, and some functionality may change before it becomes generally available.
```
Comment (kdaswani) [inline src/connections/destinations/catalog/actions-adobe-target-web/index.md:15]:
```suggestion
> The Adobe Target Web destination is in beta and is in active development. Some functionality may change before it becomes generally available.
```

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 3]
```diff
@@ -2,13 +2,14 @@
...
-hidden: true
-
-beta: true
+strat: salesforce
 id: 61957755c4d820be968457de
 ---
 Segment’s Salesforce (Actions) destination allows you to create, update or upsert records for any object type. Segment sends data to the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_rest.htm){:target="_blank"}. 
 
+> info ""
+> This document is about a feature which is in beta. This means that the Salesforce (Actions) destination is in active development, and some functionality may change before it becomes generally available.
```
Comment (kdaswani) [inline src/connections/destinations/catalog/actions-salesforce/index.md:11]:
```suggestion
> The Salesforce (Actions) destination is in beta and is in active development. Some functionality may change before it becomes generally available.
```

File: src/connections/destinations/catalog/actions-adobe-target-cloud/index.md [hunk 4]
```diff
@@ -68,15 +70,15 @@ You can use Profile Attributes in the Adobe Target Audience builder to construct
...
+Adobe Target Audiences can be used in Activities, such as A/B Testing and Experience Targeting. Please note that while Standard and Premium Adobe Target packages allow access to the SDK and API, certain personalization functionality may only be available with Adobe Target Premium. 
 
 ## FAQ
 ### Why am I getting a `Profile Not Found` error?
-The Adobe Target API can only be used for profile updates. Profiles must be created on the web first, either using Segment’s [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/) or a native implementation of `at.js`. Please ensure you are creating profiles via `at.js` first.
+The Adobe Target API can only be used for profile updates. Profiles must be created on the web first, either using Segment’s [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/) or a native implementation of `at.js`. Please ensure you are creating profiles on the web first.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-cloud/index.md:80]:
```suggestion
The Adobe Target API can only be used for profile updates. You must first create profiles on the web by using either Segment’s [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/) or a native implementation of `at.js`. Please ensure you create profiles on the web first.
```

File: src/connections/destinations/catalog/actions-adobe-target-cloud/index.md [hunk 5]
```diff
@@ -12,6 +11,9 @@ Segment offers two destinations for Adobe Target:
 - [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
 - [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)
 
+> info ""
+> The Adobe Target Cloud Mode destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
 > success ""
 > **Good to know**: This page is about Segment's Adobe Target Cloud Mode destination. There's also a page about Segment's [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/). **In order to use Adobe Target Cloud Mode, you must have a parallel web integration with Adobe Target as profiles can only be created by the Adobe Target `at.js` web script.**
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-cloud/index.md:18]:
```suggestion
> This page is about Segment's Adobe Target Cloud Mode destination. There's also a page about Segment's [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/). **In order to use Adobe Target Cloud Mode, you must have a parallel web integration with Adobe Target as profiles can only be created by the Adobe Target `at.js` web script.**
```

File: src/connections/destinations/catalog/actions-adobe-target-cloud/index.md [hunk 6]
```diff
@@ -12,6 +11,9 @@ Segment offers two destinations for Adobe Target:
 - [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
 - [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)
 
+> info ""
+> The Adobe Target Cloud Mode destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
 > success ""
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-cloud/index.md:17]:
```suggestion
> success "Good to know"
```

File: src/connections/destinations/catalog/actions-adobe-target-web/index.md [hunk 7]
```diff
@@ -2,16 +2,18 @@
...
+Adobe Target is the A/B testing and personalization component of Adobe Experience Cloud. Segment’s Adobe Target integration enables customers to send data from Segment to Adobe Target to create and update user profiles. You can leverage these profiles in Adobe Target to construct audiences and personalize onsite visitor experiences.
 
 Segment offers two destinations for Adobe Target:
 - [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
 - [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)
 
+> info ""
+> The Adobe Target Web destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
 > success ""
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-web/index.md:17]:
```suggestion
> success "Good to know"
```

File: src/connections/destinations/catalog/actions-adobe-target-web/index.md [hunk 8]
```diff
@@ -2,16 +2,18 @@
...
 > **Good to know**: This page is about Segment's Adobe Target Web destination. There's also a page about Segment's [Adobe Target Cloud Mode destination](/docs/connections/destinations/catalog/actions-adobe-target-cloud/).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-adobe-target-web/index.md:18]:
```suggestion
> This page is about Segment's Adobe Target Web destination. There's also a page about Segment's [Adobe Target Cloud Mode destination](/docs/connections/destinations/catalog/actions-adobe-target-cloud/).
```

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 9]
```diff
@@ -2,13 +2,14 @@
...
-beta: true
+strat: salesforce
 id: 61957755c4d820be968457de
 ---
 Segment’s Salesforce (Actions) destination allows you to create, update or upsert records for any object type. Segment sends data to the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_rest.htm){:target="_blank"}. 
 
+> info ""
+> The Salesforce (Actions) destination is in beta and is in active development. Some functionality may change before it becomes generally available.
+
 > success ""
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-salesforce/index.md:13]:
```suggestion
> success "Good to know"
```

File: src/connections/destinations/catalog/actions-salesforce/index.md [hunk 10]
```diff
@@ -2,13 +2,14 @@
...
 > **Good to know**: This page is about the [Actions-framework](/docs/connections/destinations/actions/) Salesforce destination. There's also a page about the [non-Actions Salesforce destination](/docs/connections/destinations/catalog/salesforce/). Both of these destinations receive data _from_ Segment.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-salesforce/index.md:14]:
```suggestion
> This page is about the [Actions-framework](/docs/connections/destinations/actions/) Salesforce destination. There's also a page about the [non-Actions Salesforce destination](/docs/connections/destinations/catalog/salesforce/). Both of these destinations receive data _from_ Segment.
```

File: src/connections/destinations/catalog/salesforce/index.md [hunk 11]
```diff
@@ -3,11 +3,15 @@ title: Salesforce Destination
...
-> info ""
-> Segment is aware of Salesforce's plans to enforce multi-factor authentication in 2022, and is evaluating solutions to ensure uninterrupted connectivity with your Salesforce account.
 
 Segment's Salesforce destination allows you to identify leads without using SOAP APIs.
 
+> info ""
+> Segment is aware of Salesforce's plans to enforce multi-factor authentication in 2022, and advises migrating to our new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/) which supports OAuth 2.0.
+
+> success ""
+> **Good to know**: This page is about the classic Salesforce Segment destination. There's also a page about the new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/). Both of these destinations receive data _from_ Segment. We recommend using the new Salesforce (Actions) destination for additional functionality and flexibility.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/salesforce/index.md:13]:
```suggestion
> success "Good to know"
> This page is about the classic Salesforce Segment destination. There's also a page about the new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/). Both of these destinations receive data _from_ Segment. We recommend using the new Salesforce (Actions) destination for additional functionality and flexibility.
```

**Review by stayseesong — APPROVED** [review]

File: src/connections/destinations/catalog/salesforce/index.md [hunk 12]
```diff
@@ -3,11 +3,15 @@ title: Salesforce Destination
...
-> info ""
-> Segment is aware of Salesforce's plans to enforce multi-factor authentication in 2022, and is evaluating solutions to ensure uninterrupted connectivity with your Salesforce account.
 
 Segment's Salesforce destination allows you to identify leads without using SOAP APIs.
 
+> info ""
+> Segment is aware of Salesforce's plans to enforce multi-factor authentication in 2022, and advises migrating to our new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/) which supports OAuth 2.0.
+
+> success "Good to know"
+> This page is about the classic Salesforce Segment destination. There's also a page about the new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/). Both of these destinations receive data _from_ Segment. We recommend using the new Salesforce (Actions) destination for additional functionality and flexibility.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/salesforce/index.md:13]:
```suggestion
> This page is about the classic Salesforce Segment destination. There's also a page about the new [Salesforce (Actions) destination](/docs/connections/destinations/catalog/actions-salesforce/). Both of these destinations receive data _from_ Segment. Use the new Salesforce (Actions) destination for additional functionality and flexibility.
```

**Comment by kdaswani** [comment]
@stayseesong Are we good to merge this into develop so it goes live in tomorrow's deploy? 😄

**Pull request merged**